packages/pass-style/src/make-far.js

```diff
@@ -128,16 +129,57 @@ export const Remotable = (
 };
 harden(Remotable);
 
+/**
+ * The name of the automatically added default meta-method for obtaining a
+ * list of all methods of an object declared with `Far`, or an object that
+ * inherits from an object declared with `Far`.
+ *
+ * Modeled on `GET_INTERFACE_GUARD` from `@endo/exo`.
+ *
+ * TODO Name to be bikeshed. Perhaps even whether it is a
+ * string or symbol to be bikeshed.
```

Comment:
after this is released?

What will the impact be of changing it then?

Reply:
It won't be pleasant. This being a non-`test/` name, if you have a suggestion for a better name, the earlier we switch to it the better.

The symbol issue is tied to an ongoing ocapn controversy about whether symbols should be Passable at all. The current consensus ocapn agreement does include them, but @kriskowal has good arguments about why we shouldn't. We're got very few, but this PR does add to that. OTOH, if we are keeping symbols as Passable at all, I find it very attractive that the meta methods at stake in this PR should be symbols.

At least this PR exports these method-names as the value of importable variables. If all code using these do so only via the imported variable, that will ease the transition code. But it'll still be substantial because these become protocol between code linked against different versions.

Reply:
Leaving this conversation unresolved for now. Reviewers + @kriskowal , what do you think we should do about naming these methods?

Reply:
until there's a durable name, you could indicate that it shouldn't be relied upon (even though it's exported). e.g. by prefixing with underscores. Same for `__GET_METHOD_GUARD`.

Reply:
> you could indicate that it shouldn't be relied upon
IIUC the naming controversy is not for `export const GET_METHOD_NAMES`, it's for its value, `Symbol.for('getMethodNames')`.  The `GET_METHOD_NAMES` exported binding can be used widely in importing code without any problems, but if the value (the `Symbol.for`) is changed, there will be an incompatibility.

Having said that, @erights, this is a case where a Symbol is used to avoid unintentional conflicts with an object's existing properties.  Interestingly, leading underscores have performed this task in other situations.  I would rather not add to the ocapn controversy to keep our options open.  Why not:

```js
export const GET_METHOD_NAMES = '__getMethodNames';
export const GET_INTERFACE_GUARD = "__getInterfaceGuard';
```

Reply:
> IIUC the "bikeshed" is not for export const GET_METHOD_NAMES, it's for its value,

Correct.

> ```js
> export const GET_METHOD_NAMES = '__getMethodNames';
> export const GET_INTERFACE_GUARD = '__getInterfaceGuard';
> ```

Our more common convention is both a prefix and a suffix double underbar

```js
export const GET_METHOD_NAMES = '__getMethodNames__';
export const GET_INTERFACE_GUARD = '__getInterfaceGuard__';
```

Done.